configserver/src/main/java/com/yahoo/vespa/config/server/deploy/Deployment.java

```diff
@@ -149,6 +156,9 @@ private void restartServices(ApplicationId applicationId) {
         RestartActions restartActions = configChangeActions.getRestartActions().useForInternalRestart(internalRedeploy);
 
         if ( ! restartActions.isEmpty()) {
+
+            waitForConfigToConverge(applicationId);
```

Comment:
Who does the waiting here? I guess it could take a long time.

Reply:
Yes, will do the same as in `waitForResources()`